ds-prep-python-tuples/ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,51 @@
+# 3
+passenger = tuple((12, True, 'Bonnell, Miss. Elizabeth', 'female', 58))
```

Comment:
`(12, True, 'Bonnell, Miss. Elizabeth', 'female', 58)` is already `tuple` before passing it to the `tuple` function: https://www.w3schools.com/python/python_tuples.asp